lib/Interpreter/ValueExtractionSynthesizer.h

```diff
@@ -90,7 +90,7 @@ namespace cling {
 
     // Find and cache cling::runtime::gCling, setValueNoAlloc,
     // setValueWithAlloc on first request.
-    void FindAndCacheRuntimeDecls();
+    bool FindAndCacheRuntimeDecls(clang::Expr*);
```

Comment:
This will conflict with something else, if the conflicting PR is denied I can handle it then.